src/main/java/org/skife/jdbi/v2/sqlobject/SqlObject.java

```diff
@@ -187,9 +187,7 @@ else if (mixinHandlers.containsKey(raw_method)) {
         // this is an implicit mixin, not an explicit one, so we need to *always* add it
         handlers.putAll(CloseInternalDoNotUseThisClass.Helper.handlers());
 
-        handlers.putAll(EqualsHandler.handler());
         handlers.putAll(ToStringHandler.handler(sqlObjectType.getName()));
```

Comment:
I'm wondering why we don't remove `ToStringHandler`. Do we expect that someone could override it?